misk-aws2-dynamodb-testing/build.gradle.kts

```diff
@@ -25,6 +25,10 @@ dependencies {
   implementation("org.apache.logging.log4j:log4j-core:2.18.0")
   implementation("org.apache.logging.log4j:log4j-api:2.18.0")
 
+  // There is in incompatibility between com.amazonaws:DynamoDBLocal and recent ANTLR versions.
+  // We have to force down to a known-working version.
+  implementation("org.antlr:antlr4-runtime") { version { strictly("4.9.3") }}
```

Comment:
Note that https://github.com/cashapp/misk/commit/12cb717a6e46fd6226feeabdc3f9fe8f555112ae added a check so you can remove these clamps

Reply:
Thanks Tristan, reverted. And thanks for getting the buildkite stuff sorted 🙏

Reply:
no problem! feel free to merge whenever

Reply:
lol I can't, no write access. Are you able to merge/approve the test runs?

Reply:
I've pressed the buttons.